packages/remark-shaku-code-annotate/src/index.ts

```diff
@@ -299,6 +310,18 @@ function parseLines(lines: IThemedToken[][]) {
         } as const;
       }
     }
+    
+    if (line.length > 0 && isShaKuLine) {
+      isShaKuLine = false;
+      const commentBlockEnd = /^(?<leadingSpaces>\s*)\*\//.test(line[0].content);
```

Comment:
this assumes all comment block ends with `*/`, which is not solid.
also it might be better put in `parseComment()`